runatlantis.io/docs/pre-workflow-hooks.md

```diff
@@ -4,27 +4,29 @@ Pre workflow hooks can be defined to run scripts before default or custom
 workflows are executed. Pre workflow hooks differ from [custom
 workflows](custom-workflows.html#custom-run-command) in several ways.
 
-1. Pre workflow hooks do not require for repository configuration to be
+1. Pre workflow hooks do not require the repository configuration to be
    present. This can be utilized to [dynamically generate repo configs](pre-workflow-hooks.html#dynamic-repo-config-generation).
 2. Pre workflow hooks are run outside of Atlantis commands. Which means
    they do not surface their output back to the PR as a comment.
-3. Pre workflow hooks only allow `run` and `description` commands.
 
 [[toc]]
 
 ## Usage
-Pre workflow hooks can only be specified in the Server-Side Repo Config under
-`repos` key. 
+
+Pre workflow hooks can only be specified in the Server-Side Repo Config under the
+`repos` key.
 ::: tip Note
-`pre-workflow-hooks` do not prevent Atlantis from executing its
+`pre-workflow-hooks` do not prevent Atlantis from executing it's
 workflows(`plan`, `apply`) even if a `run` command exits with an error.
 ::: 
 
 ## Use Cases
+
 ### Dynamic Repo Config Generation
-If you want generate your `atlantis.yaml` before Atlantis can parse it. You
+
+If you want to generate your `atlantis.yaml` before Atlantis can parse it, you
```

Comment:
Since we're updating this line, might as well drop the "you"

```suggestion
To generate the repo `atlantis.yaml` before Atlantis can parse it,
```

Reply:
Done